src/games/kalambury/Board/GameBoard.js

```diff
@@ -1,29 +1,15 @@
 import React, { useEffect, useState } from "react";
 import { Header } from "semantic-ui-react";
 import Confetti from "react-dom-confetti";
-import { shuffle } from "lodash";
 
 import { timerFormat, currentTime } from "../utils/time";
-import { COLORS } from "config/constants";
+import { CONFETTI_CONFIG } from "config/constants";
```

Comment:
There is also confetti in picturematch game, but it uses different config. Maybe we should keep a file with a few predefined configs? (something like WIDE_CONFETTI, HORIZONTAL_CONFETTI etc)